lighthouse-cli/test/fixtures/a11y/a11y_tester.html

```diff
@@ -197,5 +197,13 @@
     <section>
       <video id="video-description"></video>
     </section>
+    <script>
+      // axe fails if `define` is present but misbehaves, create a bogus implementation to exercise our
```

Comment:
I don't totally get this. I'll ask you in person if i'm interpreting it correctly, but it could probably use a slightly more extended comment. :)

Reply:
```
// axe fails if a different UMD loader like almond.js is used on the page, see https://github.com/GoogleChrome/lighthouse/issues/2505
// We can simulate this behavior by defining a similarly misbehaving `define` function.
```

sounds better?

Reply:
YESSS so good